src/networks/evm/index.ts

```diff
@@ -10,19 +10,21 @@ import { NetworkID, Space } from '@/types';
 const METADATA = {
   gor: {
     name: 'Ethereum Goerli',
-    chainId: 5
+    chainId: 5,
+    apiUrl: 'https://api.studio.thegraph.com/query/41343/sekhmet-sx-goerli/version/latest'
```

Comment:
@bonustrack could you share links to our Studio URLs? They have some non-public IDs.

Reply:
sx-goerli https://api.studio.thegraph.com/query/23545/sx-goerli/version/latest
sx-sepolia https://api.studio.thegraph.com/query/23545/sx-sepolia/version/latest

Reply:
There isn't any "production deployment" with API key for these, I only see such URL on the mainnet subgraph Studio, probably testnet don't need such keys